### DISCUSSION TITLE
Recommendations for setting up the output display

### DISCUSSION THREAD

**Discussion by Vizard-Zh** [LizardByte/community#381] [Sunshine]
The output name is not fixed on Windows, for example, after the lid of the laptop is closed, the output name of the external monitor may become \\.\DISPLAY1.
In the current version of Sunshine, if I want to specify the content on the monitor 2, I can only set the Output Name to \\.\DISPLAY1 in Sunshine-Configuration-Audio/Video, but if the lid is closed, this setting will not work.
Therefore, it is recommended that in the future version, other specified methods be added, such as the device ID of the monitor, the display ID of the monitor, the short monitor ID, and other fixed items to avoid abnormal problems.
Thank you very much !

**Comment by ReenigneArcher** [comment]
I believe device id is supported already.

**Reply by Squikou** [reply]
Hi,
Sorry but I have the same problem as Vizard-ZH.
You say it is possible to use Device-ID? how to do it please?

**Reply by ReenigneArcher** [reply]
Sorry, I was mistaken. It will be possible after this PR is merged. https://github.com/LizardByte/Sunshine/pull/2032
